### PR TITLE
Fix bug where credential inputs were not filled on edit

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #9310 

Just needed to set the initial input values if they are present

Screen Shot 2021-02-16 at 6 51 39 PM

You don't need to upload the gce file to recreate this.  Just create a GCE cred and then try to edit.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

File: awx/ui_next/src/screens/Credential/CredentialEdit/CredentialEdit.jsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ function CredentialEdit({ credential, me }) {
         },
       ] = await Promise.all([
         CredentialTypesAPI.read({ page_size: 200 }),
-        CredentialsAPI.readInputSources(credId, { page_size: 200 }),
```
Comment (mabashian) [inline awx/ui_next/src/screens/Credential/CredentialEdit/CredentialEdit.jsx:119]: This method only takes one param: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/api/models/Credentials.js#L23

It defaults to a page size of 200

Comment (unlikelyzero) [reply awx/ui_next/src/screens/Credential/CredentialEdit/CredentialEdit.jsx:119]: @mabashian for the record, there's a hidden performance problem here that I can't see.

**Comment by unlikelyzero** [comment]
@mabashian is there a way to add a simple unit test?

**Comment by unlikelyzero** [comment]
@mabashian i think you might want to add a unit test. Also: https://dashboard.cypress.io/projects/ui2mq7/runs/4460 is green

**Review by jakemcdermott — APPROVED** [review]

**Review by unlikelyzero — APPROVED** [review]

**Pull request merged**